apps/video-importer/README.md

```
@@ -32,21 +32,48 @@ Before you can use the Video Importer binary, make sure you have the following:
+
 ---
 
 ## Supported File Types & Formatting
 
 ### Video Files
 
 - **Format:** `.mp4` only
-- **Naming Convention:**
+- **Naming Convention (classic, 4 segments):**
 
   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)